quinn/src/platform/unix.rs

```diff
@@ -167,17 +167,52 @@ impl super::UdpExt for UdpSocket {
         Ok(sent)
     }
 
+    #[cfg(not(any(target_os = "macos", target_os = "ios")))]
```

Comment:
Would be nice to do the factoring out of methods in a separate commit from introducing batching.

Reply:
Done.